wrims_v2/wrims_v2/test/data/README.md

```diff
@@ -0,0 +1 @@
+DSS files generated by test code go here
```

Comment:
Recommend this goes in a directory that is gitignored and gets cleaned. When integrating into gradle that would be under the build directory